modules/gdscript/gdscript_parser.h

```diff
@@ -223,7 +223,7 @@ class GDScriptParser {
 		}
 
 		bool operator!=(const DataType &p_other) const {
-			return !(this->operator==(p_other));
+			return !(operator==(p_other));
```

Comment:
```suggestion
			return !(*this == p_other);
```
Already added but will update after CI has run